scripts/observers/config.py

```diff
@@ -125,6 +141,47 @@ def prepare_seed_node(working_dir):
     os.chmod(working_dir/"seednode/seednode", st.st_mode | stat.S_IEXEC)
 
 
+def prepare_proxy(working_dir):
+    print("preparing proxy")
+    mx_chain_proxy_go_folder = working_dir / "mx-chain-proxy-go"
+    subprocess.check_call(["go", "build"], cwd=mx_chain_proxy_go_folder / "cmd/proxy")
+
+    mx_chain_proxy_go_binary_folder = mx_chain_proxy_go_folder / "cmd/proxy"
+    st = os.stat(mx_chain_proxy_go_binary_folder / "proxy")
+    os.chmod(mx_chain_proxy_go_binary_folder / "proxy", st.st_mode | stat.S_IEXEC)
+
+    # prefs.toml
```

Comment:
wrong comment? you don't seem to touch `prefs.toml` here

Reply:
changed